### ISSUE TITLE
Not finding all the accounts

### ISSUE THREAD

**Issue by maximousblk** [soxoj/maigret#102]
The number of accounts found is less than half of what Sherlock shows

**Comment by soxoj** [comment]
Thank you for your feedback. Could you please also share username example or full results of both Maigret and Sherlock so I can determine the reason?

**Comment by maximousblk** [comment]
I am testing on my own username `maximousblk`

When running locally, I get
Sherlock: 104 results
Maigret: 43 results

And when using the telegram bot, I get 105 results

**Comment by soxoj** [comment]
Looks like it's a network-caused problems -- I've got 73 results for your username for the first run with default timeout (10 sec) and 99 results for the second run with increased timeout (30 sec).
The telegram bot are using 20 sec timeout (and stable network connection), and Sherlock are not using timeout at all.

I'm going to implement session scan statistics feature in the neat future ([roadmap](https://github.com/soxoj/maigret/wiki/Roadmap)). It should report to user the amount of checks with timeouts and give an advice to increase timeout manually. Also I'm going to implement retries someday.

**Long story short**: run with `--timeout` and good enough seconds for maximum results.

**Comment by maximousblk** [comment]
Increasing the timeout does show more results but too much timeout causes #103
